k8s/template_manager.go

```diff
@@ -286,6 +286,8 @@ func (tm *TemplateManager) Run(ctx context.Context, template *templatev1.Templat
 			tm.Log.Error(err, "failed to set condition on resource", "kind", source.GetKind(), "name", source.GetName(), "namespace", source.GetNamespace(), "conditionValue", conditionValue)
 		}
 	}
+
+	tm.Log.V(2).Info("Reconcile Complete", "template", template.Name)
```

Comment:
this will fill up the logs pretty quickly and doesn't really add much info ?
```suggestion
```

Reply:
V(2) logs are debug only and will not appear in normal operation. This was used during debugging to check how long the reconcile loops were taking to execute, along with similar output that was already in the schema manager: 		https://github.com/flanksource/template-operator/blob/d09cbe1e75a6d836668fc3e6147ec480902341af/k8s/schema_cache.go#L46
I can remove it if you like, but it may be useful for debug in the future.

Reply:
Before / After type log statements are generally only useful during development where they can easily be added in, but come at a huge cognitive cost for operators needing to sift through details. 

An alternative is to move this to V(3) and reduce log level in the args to V(2)

Reply:
log level in args is v1 by default, but I've bumped this to v3 too